relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -193,51 +201,99 @@ fn scrub_file(description: &str) -> Option<String> {
     }
 }
 
-/// Normalize the domain and port of a URL.
+/// Scrub a `Url::Host` object. For domain names, run the string through a scrubber. For IP addresses, return the IP address as-is.
```

Comment:
Note there are multiple errors in these docs!

In this one:
- IP addresses are scrubbed.
- Return value is not an option.
- Return value is not a `String`.